### PR TITLE
Vagrantfile: enable gettys

### PR DESCRIPTION
**Pull request by rstyd**
Addresses #840

### CODE REVIEW

**Review by reidpr — CHANGES_REQUESTED** [review]
Great that it was so short! A few suggestions:

1. Do we need five extra? I recall two, but I think actually it's just one because it was whatever was on TTY1 (which is now the autologin stuff) plus one for the test suite.
2. Document briefly why we are enabling the getty(s) here in the Vagrantfile.
3. I was pretty sure this used to be explained in the documentation somewhere? But, it needs to be documented. Even though it's a pedantic fail, we don't want to go on this chase again. Perhaps a comment at the point the test fails?

**Comment by rstyd** [comment]
1. We do not. We could get away with just TTY2. I was thinking we could just enable them all since that seems to be the normal configuration Linux installations have. 
2. Will do. 
3. I'll add something to the test output.

File: packaging/vagrant/Vagrantfile [hunk 1]
```diff
@@ -294,6 +294,10 @@ EOF2
     cp /lib/systemd/system/getty@.service getty@tty1.service
     sed -ri 's|^ExecStart=.*$|ExecStart=-/sbin/agetty --autologin charlie --noclear %I|' getty@tty1.service
 
+    # We need to enable another TTY for the tests to pass
+    # One test attempts to signal this getty process from within the container
+    sudo systemctl enable getty@tty2.service
```
Comment (reidpr) [inline packaging/vagrant/Vagrantfile:299]: This section doesn't need sudo. Could also fix the other similar errors.

**Review by reidpr — APPROVED** [review]
Looks good pending your approval of my tidy. Thanks!

**Comment by rstyd** [comment]
Looks good to me!

**Pull request merged**